python/ml4ir/applications/ranking/model/losses/listwise_losses.py

```diff
@@ -59,20 +69,20 @@ def get_final_activation_op(self, output_name):
             Uses `mask` field to exclude padded records from contributing
             to the softmax activation
         """
-        softmax_op = layers.Softmax(axis=-1, name=output_name)
+        softmax_op = layers.Softmax(axis=-1)
```

Comment:
are we using this? there is a note that it is more stable, but then we use `tf.where(..)`